### PR TITLE
Add FW MQB Skoda Kodiaq Mk1 2021MY

### PR DESCRIPTION
**Pull request by vladimiryuriev**

### CODE REVIEW

**Comment by vanillagorillaa** [comment]
Please include the dongleID and route that this firmware came from.

**Comment by jyoung8607** [comment]
In addition to providing the dongle ID, can you add the new firmware in sorted order? Also, can you elaborate on why you're adding two separate firmware versions for fwdRadar?

File: docs/CARS.md [hunk 1]
```diff
@@ -141,7 +141,7 @@
 | SEAT      | Leon 2014-2020                | Driver Assistance | Stock            | 0mph               | 0mph         |
 | Škoda     | Kamiq 2021<sup>2</sup>        | Driver Assistance | Stock            | 0mph               | 0mph         |
 | Škoda     | Karoq 2019                    | Driver Assistance | Stock            | 0mph               | 0mph         |
-| Škoda     | Kodiaq 2018-19                | Driver Assistance | Stock            | 0mph               | 0mph         |
+| Škoda     | Kodiaq 2018-19, 2021          | Driver Assistance | Stock            | 0mph               | 0mph         |
```
Comment (jyoung8607) [inline docs/CARS.md:144]: comma allows us to fill single-year support gaps as long as we're confident the missing year will work, and I am.

```suggestion
| Škoda     | Kodiaq 2018-21                | Driver Assistance | Stock            | 0mph               | 0mph         |
```

**Comment by adeebshihadeh** [comment]
Closing due to inactivity. Feel free to reopen once you can provide a route.

**Pull request closed without merging**